src/generateMarkdown.ts

```diff
@@ -74,7 +74,7 @@ ${type}${arraySuffix}
 </td>
 <td>
 
-${field.description}
+${field.docsLink ? `<a href="${field.docsLink}">${field.description}</a>` : field.description}
```

Comment:
I might make this a separate link rather than making the whole description a link